jiant/tasks/core.py

```diff
@@ -176,6 +176,12 @@ def collate_fn(cls, batch):
             raise TypeError(f"Unknown type for collate_fn {type(elem)}")
 
 
+class SuperGlueMixin:
+    @classmethod
+    def super_glue_format_preds(cls, pred_dict):
```

Comment:
Briefly describe why this exists/where it gets used? (Mind adding for GlueMixin as well?)